src/app/app.component.ts

```diff
@@ -67,21 +68,15 @@ export class AppComponent implements OnInit, AfterViewInit {
    */
   showClose: boolean;
 
-  constructor(
-    private config: ConfigService,
-    private userService: UserService,
-    public testSupport: TestSupportService,
-    private notificationService: NotificationService,
-    private nav: NavigationService,
-    private modalService: ModalService,
-  ) {}
+  constructor(private config: ConfigService,
+              private userService: UserService,
+              public testSupport: TestSupportService,
+              private notificationService: NotificationService,
+              private nav: NavigationService,
+              private modalService: ModalService,
+              public tourService: TourService) {}
 
   ngOnInit() {
-    this.productBuild = this.config.getSettings(
-      'branding',
-      'productBuild',
-      false,
-    );
```

Comment:
erm, why are these lines being removed?

Reply:
This must have been removed when resolving a merge conflict; I'll add them again now, sorry!